cpp/ps0002/cpp_ps0002.cpp

```diff
@@ -1,42 +1,37 @@
-/* Give examples for variables from real life and consider that:
-    - variables should be named in a way that makes sense
-    - variables should has a suitable data type
-    - variables should be initialized with a suitable value
-    - example:
-        int family_members = 8;
-*/
-
-
-
 #include <iostream>
 
 int main() {
     // Variables from real life-----
 
     // 1- Number of residents in a building
-    unsigned short int buildingResidents = 80;
+    unsigned short int buildingResidents = 80; // 0.75/0.75
 
     // 2- Number of alternatives for a drug in a pharmacy
-    unsigned short int drugAlternatives = 6;
+    unsigned short int drugAlternatives = 6; // 0.75/0.75
 
     // 3- Number of team members in a programming company
-    unsigned short int teamMembers = 12;
+    unsigned short int teamMembers = 12; // 0.75/0.75
 
     // 4- Price of a bakery item
-    float bakeryItemPrice = 3.5f;
+    float bakeryItemPrice = 3.5f; // 0.75/0.75
 
     // 5- First letter of a person's name
-    char firstLetter = 'M';
+    char firstLetter = 'M'; // 0.75/0.75
 
     // 6- Number of births in a large population country like China
-    unsigned long chinaBirths = 18000000L;
+    int chinaBirths = 18000000; // 0.5/0.75
 
     // 7- More accurate result of dividing a number by 7
-    double divisionResult = 20.0 / 7.0;
+    double divisionResult = 20.0 / 7.0; // 0.75/0.75
 
     // 8- Electricity bill for a production line in Egypt
-    double electricityBill = 15000.0;
+    double electricityBill = 15000.0; // 0.75/0.75
 
     // 9- Age of people (young and old) in seconds
-    unsigned long long ageInSeconds = 743098621LL;
+    unsigned int ageInSeconds = 743098621; // 0.5/0.75
+
+    // total 6.25/6.75
+
+    return 0;
 }
+// great work 😎 10/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code